File: openapi/code_samples/python/api_company_{cui}_financial_indicators_summary.py
from flask import Flask, request, jsonify

app = Flask(__name__)

@app.route('/api/company/summary/<string:cui>', methods=['GET'])
def get_summary(cui):
    currency = request.args.get('currency', 'RON')
    # Your implementation here
    return jsonify({}), 200

if __name__ == '__main__':
    app.run()

File: openapi/code_samples/csharp/api_company_{cui}_financial_indicators_summary.cs
using Microsoft.AspNetCore.Mvc;
using Swashbuckle.AspNetCore.Annotations;

[ApiController]
[Route("api/[controller]")]
public class CompanyController : ControllerBase
{
    [HttpGet]
    [Route("summary/{cui}")]
    [SwaggerOperation(Summary = "Retrieves the summary of a given company CUI.")]
    [ProducesResponseType(typeof(SummaryReportResponseDTO), 200)]
    [ProducesResponseType(typeof(Fault), 400)]
    public IActionResult GetSummary([FromRoute] string cui, [FromQuery(Name = "currency")] string currency = "RON")
    {
        // Your implementation here
        return Ok();
    }
}
